mne/io/write.py

```diff
@@ -18,6 +18,13 @@
 from ..externals.six import string_types, b
 
 
+# We choose a "magic" date to store (because meas_date is obligatory)
+# to treat as meas_date=None. This one should be impossible for systems
+# to write -- the second field is microseconds, so anything >= 1e6
+# should be moved into the first field (seconds).
+DATE_NONE = (0, 2 ** 31 - 1)
```

Comment:
maybe overkill but do you think it makes sense to add a warning or error when reading a file if the microsecond field is >= 1e6 and not 2 ** 31 - 1 ?

Reply:
Probably overkill, if it happens nothing will really break anyway

Reply:
yes, that's true ... :)